externals/kyuubi-spark-sql-engine/src/main/scala/org/apache/kyuubi/engine/spark/events/JsonEventLogger.scala

```diff
@@ -49,8 +51,9 @@ class JsonEventLogger(logName: String, hadoopConf: Configuration)
   private val writers = HashMap.empty[String, Logger]
 
   private def getOrUpdate(event: KyuubiEvent): Logger = synchronized {
-    writers.getOrElseUpdate(event.eventType, {
-      val eventPath = new Path(new Path(logRoot), event.eventType)
+    val partitions = event.partitions.map(kv => s"${kv._1}=${kv._2}").mkString("/")
+    writers.getOrElseUpdate(event.eventType + partitions, {
+      val eventPath = new Path(new Path(new Path(logRoot), event.eventType), partitions)
```

Comment:
new Path(new Path("abc"), "")
java.lang.IllegalArgumentException: Can not create a Path from an empty string